README.md

```diff
@@ -8,13 +8,13 @@ This repository contains the following packages [^fn1]:
 
 <!-- start package list -->
 
-- [`@metamask/accounts-controller`](packages/accounts-controller)
```

Comment:
🤔 What's going on with the AccountsController and TransactionController changes in this file?

Reply:
not sure, but i'll fix this so only caip-stream is added

Reply:
done here https://github.com/MetaMask/core/pull/4399/commits/be35af82614bf1d9468e0ac3886f89b31b216e14